server/src/main/java/org/opensearch/index/store/lockmanager/RemoteStoreMetadataLockManager.java

```diff
@@ -75,7 +76,7 @@ public void release(LockInfo lockInfo) throws IOException {
         }
     }
 
-    public String fetchLock(String filenamePrefix, String acquirerId) throws IOException {
+    public String fetchLockedMetadata(String filenamePrefix, String acquirerId) throws IOException {
```

Comment:
The renaming was already done as part of https://github.com/opensearch-project/OpenSearch/pull/11987. Please rebase latest `main`